### ISSUE TITLE
A card with an empty content causes NPE on mouse hover

### ISSUE THREAD

**Issue by asapha** [JetBrains/compose-jb#1027]
When using this sample, an exception will be thrown when you hover the mouse over the window, rendering the app non responsive. Having the empty card deep in the tree results in the same thing.
```kotlin
fun main() = application {
    Window(
        onCloseRequest = ::exitApplication,
    ) {
        Button(onClick = {}) {
            Text("This button can't be pressed")
        }
        Card {  }
    }
}
```

```java
Exception in thread "AWT-EventQueue-0" java.lang.NullPointerException
	at ui.desktop@1.0.0-alpha3/androidx.compose.ui.platform.SkijaLayer.isInLayer-k-4lQ0M(SkijaLayer.desktop.kt:126)
	at ui.desktop@1.0.0-alpha3/androidx.compose.ui.node.LayoutNodeWrapper.withinLayerBounds-k-4lQ0M(LayoutNodeWrapper.kt:602)
	at ui.desktop@1.0.0-alpha3/androidx.compose.ui.node.DelegatingLayoutNodeWrapper.hitTest-3MmeM6k(DelegatingLayoutNodeWrapper.kt:75)
	at ui.desktop@1.0.0-alpha3/androidx.compose.ui.node.DelegatingLayoutNodeWrapper.hitTest-3MmeM6k(DelegatingLayoutNodeWrapper.kt:77)
	at ui.desktop@1.0.0-alpha3/androidx.compose.ui.node.DelegatingLayoutNodeWrapper.hitTest-3MmeM6k(DelegatingLayoutNodeWrapper.kt:77)
	at ui.desktop@1.0.0-alpha3/androidx.compose.ui.node.DelegatingLayoutNodeWrapper.hitTest-3MmeM6k(DelegatingLayoutNodeWrapper.kt:77)
	at ui.desktop@1.0.0-alpha3/androidx.compose.ui.node.LayoutNode.hitTest-3MmeM6k$ui(LayoutNode.kt:845)
	at ui.desktop@1.0.0-alpha3/androidx.compose.ui.node.InnerPlaceable.hitTest-3MmeM6k(InnerPlaceable.kt:145)
	at ui.desktop@1.0.0-alpha3/androidx.compose.ui.node.DelegatingLayoutNodeWrapper.hitTest-3MmeM6k(DelegatingLayoutNodeWrapper.kt:77)
	at ui.desktop@1.0.0-alpha3/androidx.compose.ui.node.DelegatingLayoutNodeWrapper.hitTest-3MmeM6k(DelegatingLayoutNodeWrapper.kt:77)
	at ui.desktop@1.0.0-alpha3/androidx.compose.ui.node.DelegatingLayoutNodeWrapper.hitTest-3MmeM6k(DelegatingLayoutNodeWrapper.kt:77)
	at ui.desktop@1.0.0-alpha3/androidx.compose.ui.node.DelegatingLayoutNodeWrapper.hitTest-3MmeM6k(DelegatingLayoutNodeWrapper.kt:77)
	at ui.desktop@1.0.0-alpha3/androidx.compose.ui.node.LayoutNode.hitTest-3MmeM6k$ui(LayoutNode.kt:845)
	at ui.desktop@1.0.0-alpha3/androidx.compose.ui.platform.DesktopOwner.onPointerEnter-k-4lQ0M$ui(DesktopOwner.desktop.kt:401)
	at ui.desktop@1.0.0-alpha3/androidx.compose.ui.platform.DesktopOwners.onMouseEntered(DesktopOwners.desktop.kt:207)
	at ui.desktop@1.0.0-alpha3/androidx.compose.ui.awt.ComposeLayer$initCanvas$2$mouseEntered$1.invoke(ComposeLayer.desktop.kt:191)
	at ui.desktop@1.0.0-alpha3/androidx.compose.ui.awt.ComposeLayer$initCanvas$2$mouseEntered$1.invoke(ComposeLayer.desktop.kt:190)
	at ui.desktop@1.0.0-alpha3/androidx.compose.ui.awt.AWTDebounceEventQueue$job$1.invokeSuspend(AWTDebounceEventQueue.desktop.kt:59)
	at kotlin.stdlib/kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.core.jvm@1.5.0/kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:316)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

Tested on:

MacOS 11.3.1
compose jb 1.0.0-alpha-3

**Comment by AshStefanOltmann** [comment]
Same for me with `1.0.0-alpha3` and `macOS 12 Beta 3`.
It worked with `0.4.0` before, so it's a regression.

Thank you, @asapha , for finding the root of the problem. I can workaround this now by setting a placeholder to prevent empty cards.

**Comment by AshStefanOltmann** [comment]
Having an empty Box inside also does not help.

My current workaround looks like this:
```
    /*
     * FIXME Workaround for https://github.com/JetBrains/compose-jb/issues/1027
     */
    if (imageBitmap.value == null) {
        Box(
            modifier = Modifier.size(1.dp)
        )
    }
```

**Comment by akurasov** [comment]
Tried with beta. Works fine on Mac 11 + Compose 1.0.0-beta5.

**Comment by igordmn** [comment]
>Works fine on Mac 11 + Compose 1.0.0-beta5.

Yes, this crash should be fixed in 1.0.0-beta5.

If it still crashes on your side, let us know.

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-5321">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
